metis-indexing/metis-indexing-service/src/test/java/eu/europeana/indexing/test/PublishingServiceTestWithMain.java

```diff
@@ -0,0 +1,161 @@
+package eu.europeana.indexing.test;

```

Comment:
Same concept here. There shouldn't be a test with main class and the urls should not be public.
Sufficient testing can be done with embedded databases or mocks.